addons/project/project.py

```diff
@@ -686,6 +686,12 @@ def onchange_user_id(self, cr, uid, ids, user_id, context=None):
             vals['date_start'] = fields.datetime.now()
         return {'value': vals}
 
+    def onchange_date_deadline(
```

Comment:
Put everything on one line, we [don't follow](https://www.odoo.com/documentation/9.0/reference/guidelines.html#pep8-options) the 80 char guidelines (here it actually reduces the readability IMHO)